packages/paste-website/src/context/SiteThemeContext.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import {Themes} from '../constants';
+
+export type SiteTheme = 'default' | 'sendgrid';
```

Comment:
I would love to create this type from the constant object `Themes` but that seems to be beyond my Typescript ability right now. 🤷‍♂

Reply:
Kind of a mouthful but:
```
const Themes = {DARK: 'dark', LIGHT: 'light'} as const // add `as const`

type ValueOf<O> = O[keyof O]
type Theme = ValueOf<typeof Themes> // 'dark' | 'light'
```

Reply:
Niiiiiiiice 😎